### DISCUSSION TITLE
Hitobjects in osu!catch's argon skin causes more frame drops than previous official skin iterations

### DISCUSSION THREAD

**Discussion by Kimitakari** [ppy/osu#21032] [Q&A] [answered]
### Type

Performance

### Bug description

Comparing with "argon" and "triangles" skins hitobjects in osu!catch, for some reason "argon" loses approx 50fps to "triangles" which is ~200fps and ~250fps respectfully. Using "classic" or any custom pro skin holds a solid frames with around 500fps thus it's playable but with "argon" it causes tearing. Surprisingly the "argon" bananas are more optimized than fruits themselves (maybe from this PR https://github.com/ppy/osu/pull/20993)

Down to screenshots are "argon", "triangles", "classic" and a pro skin "!YETI 1.7" from ![rostld](https://osu.ppy.sh/users/4365562/fruits).

### Screenshots or videos

![rundll32_2022-10-31_15-20-33](https://user-images.githubusercontent.com/36848514/199006625-12283586-4b3c-4f37-9496-ac3ef2b00305.png)
![rundll32_2022-10-31_15-20-42](https://user-images.githubusercontent.com/36848514/199006636-78152343-45bc-4352-a4fe-aa7ab7aa4025.png)
![rundll32_2022-10-31_15-20-48](https://user-images.githubusercontent.com/36848514/199006643-fbbe398e-0dc6-43c6-b9db-419958b80903.png)
![rundll32_2022-10-31_15-20-54](https://user-images.githubusercontent.com/36848514/199006654-006d03dd-79b2-4f34-aa72-65986dabb1b4.png)

### Version

2022.1031.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9900998/database.log)
[network.log](https://github.com/ppy/osu/files/9900999/network.log)
[performance.log](https://github.com/ppy/osu/files/9901010/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/9901011/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/9901012/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9901013/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/9901014/performance-update.log)

**Comment by peppy** [comment, marked as answer]
Performance and optimisation considerations are an ongoing effort. 

These new skins are done to get initial reception. Optimisation always comes later. We'll continue to improve over time.
